Dalamud/Game/ChatHandlers.cs

```diff
@@ -165,16 +171,25 @@ private void OnChatMessage(XivChatType type, uint senderId, ref SeString sender,
         if (clientState == null)
             return;
 
+        if (type == XivChatType.Notice)
+        {
+            if (!this.hasSeenLoadingMsg)
+                this.PrintWelcomeMessage();
+            
+            // FIXME (kazwolfe): This will only run on the first Notice received, meaning auto-updates won't happen
```

Comment:
I actually don't think we should change this. A lot of plugins are expecting to be updated when the character is logged in, and will show stuff like changelogs via ImGui or in chat.

Reply:
Hrm. Seems strange to defer updates like this but in the interest of compat I'll drop this for now.

Plugins imo shouldn't expect to update anywhere in particular, and I'd even argue that if we're auto-updating plugins, we shouldn't even load the old version. I'm sure this is one of those "sounds easy on paper but painful in practice" things, so it's not particularly worth digging for now.

If we *do* want auto-updates to fire after character login, we should still move this out of ChatHandlers - nothing is stopping a plugin from sending a `Notice` on the login screen (which is how we got into this mess in the first place).